nasa-gibs/worldview
# WV-2655: Further updates to OPERA Tour story steps

## Description
Made further updates to reduce the appearance of the HLS notifications (by hiding HLS layers that are not showing on the map) and refined the text for some steps. 

## How To Test

1. `git checkout wv-2655-opera-steps-updates`
2. `npm run build && npm start`
3. Open with [these URL parameters](http://localhost:3000/?v=-174.2584033868648,-77.00385795537836,141.53238355885276,78.40861436211223&l=Coastlines_15m,Reference_Features_15m,Reference_Labels_15m,OPERA_L3_Dynamic_Surface_Water_Extent-HLS_Provisional(disabled=5),HLS_L30_Nadir_BRDF_Adjusted_Reflectance(hidden),HLS_S30_Nadir_BRDF_Adjusted_Reflectance(hidden),Land_Water_Map&lg=true&tr=surface_water_extent&t=2023-09-16-T19%3A28%3A37Z). *Note that the tour story is hidden from the "Introduction to Worldview" tour story window, so it cannot be accessed from there.
Go through the story and make sure the text and map steps look good.

## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
